src/main/resources/youtube/settings/host/values/strings.xml

```diff
@@ -1067,7 +1067,8 @@ Tap and hold to undo."</string>
     <string name="revanced_preference_screen_seekbar_summary">Customize the seekbar components.</string>
 
     <string name="revanced_append_time_stamp_information_title">Append timestamp information</string>
-    <string name="revanced_append_time_stamp_information_summary_on">Information is appended to the timestamp.</string>
+    <string name="revanced_append_time_stamp_information_summary_on">"Information is appended to the timestamp.
+Tap to configure the video quality or playback speed. Tap and hold to toggle the appended information type."</string>
```

Comment:
I wasn't sure if `Tap to configure the video quality or playback speed.` should be in the summaries of `revanced_append_time_stamp_information_type_summary_on` or if they should be together with the information about tapping-and-holding.

Reply:
I think it will be fine